packages/ses-ava/src/ses-ava-test.js

```diff
@@ -104,16 +127,19 @@ const augmentLogging = (testerFunc, logger) => {
     } else if (typeof implFuncOrObj === 'object' && implFuncOrObj) {
       // Handle cases like `test(title, test.macro(...), ...)`.
       // Note that this will need updating if a future version of ava adds an alternative to `exec`.
-      implArg = freeze({
-        ...implFuncOrObj,
-        exec: wrapImplFunc(implFuncOrObj.exec),
-      });
+      implArg = { ...implFuncOrObj };
+      implArg.exec = wrapImplFunc(implFuncOrObj.exec);
```

Comment:
Since `exec` is unconditional, I'd still prefer it inline.

```suggestion
      implArg = {
        ...implFuncOrObj,
        exec: wrapImplFunc(implFuncOrObj.exec);
      };
```